### PR TITLE
fix(editable descriptions): adding indexing for editable descriptions

### PR DESCRIPTION
**Pull request by gabe-lyons**
## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: metadata-models/src/main/pegasus/com/linkedin/datajob/EditableDatajobProperties.pdl [hunk 1]
```diff
@@ -10,5 +10,12 @@ import com.linkedin.common.ChangeAuditStamps
   "name": "editableDatajobProperties"
 }
 record EditableDatajobProperties includes ChangeAuditStamps {
```
Comment (jjoyce0510) [inline metadata-models/src/main/pegasus/com/linkedin/datajob/EditableDatajobProperties.pdl:12]: Can we also rename this to EditableDataJobProperties? Didn't catch this before.

File: metadata-ingestion/src/datahub/metadata/schema_classes.py [hunk 2]
```diff
@@ -509,13 +509,13 @@ def deleted(self, value: Union[None, "AuditStampClass"]) -> None:
...
     def description(self) -> Union[None, str]:
-        # No docs available.
+        """Getter: Edited documentation of the chart """
         return self._inner_dict.get('description')  # type: ignore
     
     
     @description.setter
     def description(self, value: Union[None, str]) -> None:
-        # No docs available.
+        """Setter: Edited documentation of the chart """
```
Comment (jjoyce0510) [inline metadata-ingestion/src/datahub/metadata/schema_classes.py:518]: How did these sneak in ther!

Comment (gabe-lyons) [reply metadata-ingestion/src/datahub/metadata/schema_classes.py:518]: I added descriptions :)

File: metadata-models/src/main/pegasus/com/linkedin/datajob/EditableDatajobProperties.pdl (hunk 1 above)
Comment (gabe-lyons) [reply metadata-models/src/main/pegasus/com/linkedin/datajob/EditableDatajobProperties.pdl]: @jjoyce0510 done!

**Review by shirshanka — APPROVED** [review]
LGTM. Thanks @gabe-lyons

**Pull request merged**